src/main/kotlin/study/racing/domain/Cars.kt

```diff
@@ -0,0 +1,62 @@
+package study.racing.domain
+
+import study.racing.strategy.MoveStrategy
+import study.racing.strategy.RandomMoveStrategy
+
+class Cars private constructor(
+    private var cars: List<Car>,
+    private val strategy: MoveStrategy
```

Comment:
```suggestion
    private val strategy: MoveStrategy = RandomMoveStrategy()
```

코틀린스럽게 사용하는 방법 중에 하나는 default value를 사용하는 방법이 있죠! 요번 기회에 default value를 학습해볼까요?
default value를 학습하시면, of, from, constructor를 깔끔하게 정리하실 수 있으실거에요. 😊

Reply:
오... 프로퍼티에 기본값을 설정해주니 보조생성자를 통해서 넣어주던 전략패턴 코드란이 굳이 필요가 없어지네요.
와 감사합니다.